src/Core/StuntmanUser.cs

```diff
@@ -1,19 +1,34 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
 using System.Security.Claims;
 
 namespace RimDev.Stuntman.Core
 {
     public class StuntmanUser
     {
-        public StuntmanUser()
+        public StuntmanUser(string id, string name)
         {
+            if (id == null) throw new ArgumentNullException("id");
```

Comment:
Addressed in https://github.com/kendaleiv/stuntman/commit/d87c9230caefa736d86ece109e3cfd3144625fd5.